clean_parse_tables.py

```diff
@@ -24,7 +24,8 @@
             lines = fd.readlines()
 
         with open(path, 'wb') as fd:
-            if not lines[0].startswith("# Licensed under a 3-clause BSD style license - see LICENSE.rst"):
+            if not (lines[0].startswith("# Licensed under a 3-clause BSD style license - see LICENSE.rst")
+                    or lines[1].startswith("# Licensed under a 3-clause BSD style license - see LICENSE.rst")):
```

Comment:
You mean as an arg for the script? The problem with that is that there can be mixed cases in the tree, some files have a coding line, but other don't.